presto-server-rpm/pom.xml

```diff
@@ -81,8 +81,9 @@
             </plugin>
 
             <plugin>
-                <groupId>ca.vanzyl.maven.plugins</groupId>
+                <groupId>ca.vanzyl.provisio.maven.plugins</groupId>
                 <artifactId>provisio-maven-plugin</artifactId>
+                <version>1.0.8</version>
```

Comment:
I guess this should be modified in the root pom.xml

Reply:
Moved to property in root pom.xml